steam_wrapper/steam_wrapper.py

```diff
@@ -346,6 +354,24 @@ def enable_discord_rpc():
             os.symlink(src=src_rel, dst=dst)
 
 
+def enable_extensions(flatpak_info):
+    installed_ext_ids: t.Set[str]
+    installed_ext_ids = set(flatpak_info["app-extensions"])
+    installed_ext_ids |= set(flatpak_info["runtime-extensions"])
+    for ext_id, ext in EXTENSIONS.items():
+        for installed_ext_id in installed_ext_ids:
+            if not installed_ext_id.startswith(f"{ext_id}."):
+                continue
+            ext_basename = installed_ext_id[len(f"{ext_id}."):]
+            for env_var, subdirs in ext["append-env"].items():
+                for subdir in subdirs:
+                    dir_path = os.path.join(ext["mountpoint"], ext_basename, subdir)
+                    paths = [p for p in os.environ.get(env_var, "").split(os.pathsep) if p]
```

Comment:
Why do we need to normalize it like this rather than just add at end the sequence?

Reply:
Otherwise LGTM.

Reply:
It's easier this way - we don't need to check is the env var is set or not.

Reply:
Moved this to outer loop.